lib/handleRequest.js

```diff
@@ -108,9 +108,17 @@ function preHandlerCallback (err, request, reply) {
     return
   }
 
-  var result = reply.context.handler(request, reply)
-  if (result && typeof result.then === 'function') {
-    wrapThenable(result, reply)
+  try {
+    const result = reply.context.handler(request, reply)
```

Comment:
Is it possibile to wrap only this line into `try` block using `let` instead of `const`?

Reply:
I agree, there is no need to keep both wrapThenable and reply.send inside the catch block.